Python-packages/covidcast-py/covidcast/plotting.py

```diff
@@ -92,26 +94,27 @@ def plot(data: pd.DataFrame,
     kwargs["vmax"] = kwargs.get("vmax", meta["mean_value"] + 3 * meta["stdev_value"])
     kwargs["figsize"] = kwargs.get("figsize", (12.8, 9.6))
 
-    fig, ax = _plot_background_states(kwargs["figsize"])
+    ax = _plot_background_states(kwargs["figsize"]) if ax is None else ax
+    ax.axis("off")
     ax.set_title(f"{data_source}: {signal}, {day_to_plot.strftime('%Y-%m-%d')}")
     if plot_type == "choropleth":
         _plot_choro(ax, day_data, combine_megacounties, **kwargs)
     else:
         _plot_bubble(ax, day_data, geo_type, **kwargs)
-    return fig
+    return ax
 
 
 def plot_choropleth(data: pd.DataFrame,
                     time_value: date = None,
                     combine_megacounties: bool = True,
-                    **kwargs: Any) -> figure.Figure:
+                    **kwargs: Any) -> axes.Axes:
     """Plot choropleths for a signal. This method is deprecated and has been generalized to plot().
```

Comment:
Want to use the [deprecated directive](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated) here?

Reply:
added. This also made me realize our changelog is a bit incorrect, will make another PR to fix that.